README.md

```diff
@@ -31,7 +31,7 @@ AWS Deadline Cloud for 3ds Max is a python package that allows users to create [
 This library requires:
 
 1. 3ds Max 2024,
-1. Python 3.9 or higher; and
+1. Python 10 or higher; and
```

Comment:
```suggestion
1. Python 3.10 or higher; and
```

Time traveller? 🤣

Reply:
Roads? Where we are going we don't need roads!